wordpress-mobile/WordPress-iOS
# Remove MediaService.getMediaLibraryCount

Remove the get media library count function from `MediaService` and re-implement it in `MediaServiceRemote`. The function simply makes a series of API call, I don't see how it needs to be add to a "service" or "repository" type.

The re-implementation is quite simple comparing to the original Objective-C implementation. But since the function is used in Objective-C source code, it's unfortunate that we have to create some wrappers around it.

## Test Instructions

1. Turn off the "Native Photo Picker" feature flag.
2. Tap the site icon and choose "Change Site Icon".
3. Verify the number under "WordPress Media" is correct.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
What's described in the Test Instructions

3. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:


UI Changes testing checklist: N/A
